### PR TITLE
[capi-media-vision][ACR-1848] update deep learning based native API g…

### PR DESCRIPTION
**Pull request by daeinki**
…uide

Legacy native API have been deprecated so update them.

This patch updates below two document files,
	 face-detection.md
	 image-classification.md
and added below three document files,
	facial-landmark.md
	object-detection.md
	pose-landmark.md
and dropped one document file,
	pose-detection.md

With this patch, docs/application/native/guides/multimedia/pose-detection.md file is removed and added pose-landmark.md.

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by safir-srbd — COMMENTED** [review]
@daeinki From an initial screening of the PR, it seems that you have added and removed documents from Tizen Docs. To reflect these changes in the LNB, please edit the `toc_all.md` file.

**Comment by daeinki** [comment]
> @daeinki From an initial screening of the PR, it seems that you have added and removed documents from Tizen Docs. To reflect these changes in the LNB, please edit the `toc_all.md` file.

Got it. Thanks.

**Comment by tyoungroy** [comment]
@safir-srbd @asm-foysal 
Please update this PR with reviews.

**Comment by safir-srbd** [comment]
Closing and reopening PR to trigger staging build for reviewing.

**Comment by tyoungroy** [comment]
@safir-srbd @asm-foysal 
What do i do go to next step? 
(@daeinki )

**Comment by safir-srbd** [comment]
@tyoungroy 
Sorry for the late response. 

I will approve and merge this PR after one final check. It will be done today.

**Review by safir-srbd — COMMENTED** [review]
@daeinki 
Please make the changes

File: docs/application/toc_all.md [hunk 1]
```diff
@@ -144,9 +144,11 @@
 ##### [Overview](/application/dotnet/guides/multimedia/media-vision.md)
 ##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
 ##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
-##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
-##### [Image Recognition and Tracking](/application/dotnet/guides/multimedia/image-recognition.md)
-##### [Pose Detection](/application/dotnet/guides/multimedia/pose-detection.md)
+##### [Deep Learning Based Face Detection](/application/dotnet/guides/multimedia/face-detection.md)
+##### [Deep Learning Based Object Detection](/application/dotnet/guides/multimedia/object-detection.md)
+##### [Deep Learning Based Image Classification](/application/dotnet/guides/multimedia/image-classification.md)
+##### [Deep Learning Based Ficial Landmark](/application/dotnet/guides/multimedia/facial-landmark.md)
```
[the remark this answers is not in the document]
Comment (daeinki) [reply docs/application/toc_all.md]: Thanks. Updated after rebase. :)

File: docs/application/toc_all.md [hunk 2]
```diff
@@ -144,9 +144,11 @@
...
 ##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
 ##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
-##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
-##### [Image Recognition and Tracking](/application/dotnet/guides/multimedia/image-recognition.md)
-##### [Pose Detection](/application/dotnet/guides/multimedia/pose-detection.md)
+##### [Deep Learning Based Face Detection](/application/dotnet/guides/multimedia/face-detection.md)
+##### [Deep Learning Based Object Detection](/application/dotnet/guides/multimedia/object-detection.md)
+##### [Deep Learning Based Image Classification](/application/dotnet/guides/multimedia/image-classification.md)
+##### [Deep Learning Based Facial Landmark](/application/dotnet/guides/multimedia/facial-landmark.md)
+##### [Deep Learning Based Pose Landmark](/application/dotnet/guides/multimedia/pose-landmark.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:151]: @daeinki Sorry, I just realised that the changes in the `toc_all.md` file have been made in the dotnet part of the Visual Recognition, not the native part, which starts from Line 951. Please remove the changes here and make the changes in the native part.

**Review by safir-srbd — COMMENTED** [review]
@daeinki Sorry, I just realised that the changes in the toc_all.md file have been made in the dotnet part of the Visual Recognition, not the native part, which starts from Line 951. Please remove the changes here and make the changes in the native part.

File: docs/application/toc_all.md [hunk 3]
```diff
@@ -144,9 +144,11 @@
...
 ##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
 ##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
-##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
-##### [Image Recognition and Tracking](/application/dotnet/guides/multimedia/image-recognition.md)
-##### [Pose Detection](/application/dotnet/guides/multimedia/pose-detection.md)
+##### [Deep Learning Based Face Detection](/application/dotnet/guides/multimedia/face-detection.md)
+##### [Deep Learning Based Object Detection](/application/dotnet/guides/multimedia/object-detection.md)
+##### [Deep Learning Based Image Classification](/application/dotnet/guides/multimedia/image-classification.md)
+##### [Deep Learning Based Ficial Landmark](/application/dotnet/guides/multimedia/facial-landmark.md)
+##### [Deep Learning Based Pose Landmark](/application/dotnet/guides/multimedia/pose-landmark.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:151]:
```suggestion
##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
##### [Image Recognition and Tracking](/application/dotnet/guides/multimedia/image-recognition.md)
##### [Pose Detection](/application/dotnet/guides/multimedia/pose-detection.md)
```

**Review by safir-srbd — COMMENTED** [review]
@daeinki Please revert these changes to dotnet. They haven't been properly reverted.

**Comment by daeinki** [comment]
> @daeinki Please revert these changes to dotnet. They haven't been properly reverted.

Updated again. Thanks.

**Review by safir-srbd — APPROVED** [review]
@daeinki Thank you for committing the changes. LGTM.

**Pull request merged**